tests/test_plotting.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import pandas as pd
+# TODO: In the Omigami-core repo the test folder is found inside of the omigami folder and is called test instead of
+#  testS. Should we change that
```

Comment:
Let's move into omigami dir and keep it as tests (plural). We change later in the omigami-core to tests too